{{ .ProjectSnake }}/README.bazel.md

```diff
@@ -44,12 +44,8 @@ If the package is not already a dependency of the project, you'll have to do som
 ```shell
 # Update dependencies table to include your new dependency
 % vim pypyroject.toml
```

Comment:
```suggestion
% vim pyproject.toml
```
Typo (noticed but not introduced in this PR). In case you want to fix while you're in here.